src/serac/numerics/functional/tensor.hpp

```diff
@@ -1185,43 +1202,188 @@ SERAC_HOST_DEVICE constexpr tensor<T, n> linear_solve(tensor<T, n, n> A, const t
     y        = tmp;
   };
 
-  tensor<double, n> x{};
+  auto U = A;
+  // initialize L to Identity
+  auto L = tensor<T, n, n>{};
+  // This handles the case if T is a dual number
+  // TODO: consider making a dense identity that is templated on type
+  // BT 05/09/2022
```

Comment:
I don't like leaving stamps in code as this is handled by git, but I defer to @white238 's judgement.

Reply:
I don't personally put my initials on TODO's normally, but I know others prefer to.  Please just do something like:

```
TODO - BT: some comment
```

Reply:
I'm happy to follow whatever convention there is. Removing the timestamp and formatting as in Chris's comment.